### PR TITLE
[Back-end] Create Readme.md

### PR DESCRIPTION
**Pull request by nelvson**
This text, although a long one, does not necessarily have any particular meaning.

### CODE REVIEW

**Comment by RaymondAnggara** [comment]
Hmmm?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Huh?
```
Comment (greensirius) [inline README.md]: Hoh?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+Hello there, general kenobi!
```
Comment (greensirius) [inline README.md]: Hello juga

File: README.md (hunk 1 above)
Comment (andreirawan97) [inline README.md]: This is bad.

Comment (nelvson) [reply README.md]: 🤔

File: README.md (hunk 2 above)
Comment (cindmichelle) [inline README.md]: 👽

File: README.md (hunk 1 above)
Comment (greensirius) [reply README.md]: It was a useless PR btw

Comment (andreirawan97) [reply README.md]: Heh?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Huh?
+
```
[the remark this answers is not in the document]
Comment (nelvson) [reply README.md]: 🤔

**Pull request merged**